i18n/zh-CN/docusaurus-plugin-content-docs/current/release-notes-1.1.2.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Release Notes 1.1.2-RC1
+title: Release Notes 1.1.2
 sidebar_position: 0.4
 --- 
 
 Apache Linkis(incubating) 1.1.2 包括所有 [Project Linkis-1.1.2](https://github.com/apache/incubator-linkis/projects/20)。
 
 
```

Comment:
This file should not appear in the change file list.

Reply:
Done